web/components/admin/TextFieldWithSubmit.tsx

```diff
@@ -134,15 +135,17 @@ export const TextFieldWithSubmit: FC<TextFieldWithSubmitProps> = ({
           <div className="field-tip">{tip}</div>
           <FormStatusIndicator status={status || submitStatus} />
           <div className="update-button-container">
-            <Button
-              type="primary"
-              size="small"
-              className="submit-button"
-              onClick={handleSubmit}
-              disabled={!hasChanged}
-            >
-              Update
-            </Button>
+            {fieldName !== 'adminPassword' && (
```

Comment:
This feels like it shouldn't have a specific use case hard-coded in it.  Maybe if you're conditionally adding a button it can have a new property added to it that's like `hasSubmitButton` or something that can default to `true` but be set to `false` in the adminPassword case.

Reply:
I created a new `useState` isPwdInput boolean variable for this button and added a new `useEffect` function to check the `fieldName` and if it is the password input field would be set to true, else false. Please let me know if this change would resolve this code change request.